web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -149,29 +149,29 @@ The guide consists of two big steps:
 1. Migrating your Wasp project to the new structure.
 2. Migrating Wasp Auth.
 
-If you get stuck at any point, don't hesitate to ask us on [our Discord server](https://discord.gg/rzdnErX).
+If you get stuck at any point, don't hesitate to ask for help on [our Discord server](https://discord.gg/rzdnErX).
 
 ### Migrating your project to the new structure
 
 You can easily migrate your old Wasp project to the new structure by following a
 series of steps. Assuming you have a project called `foo` inside the
 directory `foo`, you should:
 
-0. Install the latest `0.12.x` version of Wasp.
+0. **Install the latest `0.12.x` version** of Wasp.
```

Comment:
In most of the important steps, I bolded a single crucial sequence of words, to make it easier to quickly read / visually anchor.